numpy/core/src/multiarray/ctors.c

```diff
@@ -2335,6 +2339,23 @@ PyArray_FromInterface(PyObject *origin)
         else {
             base = origin;
         }
+#if defined(NPY_PY3K)
+        if (PyObject_GetBuffer(base, &view, PyBUF_WRITABLE) < 0) {
```

Comment:
I know it's not necessary, but I'd find this clearer with a `| PyBUF_SIMPLE` here

Reply:
`PyBUF_SIMPLE` is just zero. The documentation already sets out the hierarchy, the macros are not orthogonal.

Reply:
I'd argue that `SIMPLE` is orthogonal to `WRITEABLE`, it just happens (/ is designed) that it's representation is `0`.

Reply:
Seems reasonable. I'll add when we decide about the comment for `PyBuffer_Release`